fl4health/feature_alignment/tabular_feature.py

```diff
@@ -10,7 +10,11 @@
 
 class TabularFeature:
     def __init__(
-        self, feature_name: str, feature_type: TabularType, fill_value: Optional[Scalar], metadata: MetaData = []
+        self,
+        feature_name: str,
+        feature_type: TabularType,
+        fill_value: Optional[Scalar],
+        metadata: Optional[MetaData] = None,
```

Comment:
Not that this is to avoid a common ["gotcha" in python for mutable collection defaults](https://docs.python-guide.org/writing/gotchas/). We just default this to None instead of an empty list and set it explicitly in the constructor. Python is dumb sometimes...

Reply:
Great catch! I didn't know about this at all 😂